src/Response.php

```diff
@@ -43,7 +43,7 @@ class Response
     /**
      * Class constructor used to construct a response
      *
-     * @param string $status
+     * @param bool|null $status
```

Comment:
`bool`? :thinking: 

Is this not an HTTP response status?

Reply:
Ha ha! It's even cast to a bool later on down the line